micromamba/tests/test_config.py

```diff
@@ -14,19 +16,21 @@ def config(*args):
     cmd = [umamba, "config"] + [arg for arg in args if arg]
     res = subprocess.check_output(cmd)
     if "--json" in args:
-        j = json.loads(res)
+        j = yaml.load(res, yaml.FullLoader)
```

Comment:
I think you've filed a bug for this already, right?

Reply:
Yes